### PR TITLE
Add metainfo manifest file

### PR DESCRIPTION
**Pull request by tim77**
I realized that we can add [metainfo file](https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/#_metainfo_xml_file_creation) and other users can easily discover and install **gedit-control-your-tabs** via GNOME Software for example.

### CODE REVIEW

File: controlyourtabs/gedit-control-your-tabs.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="addon">
+  <id>gedit-control-your-tabs</id>
```
Comment (jefferyto) [inline controlyourtabs/gedit-control-your-tabs.metainfo.xml]:
```suggestion
  <id>com.thingsthemselves.gedit.plugins.controlyourtabs</id>
```

File: controlyourtabs/gedit-control-your-tabs.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <extends>org.gnome.gedit.desktop</extends>
+  <name>gedit-control-your-tabs</name>
```
Comment (jefferyto) [inline controlyourtabs/gedit-control-your-tabs.metainfo.xml]:
```suggestion
  <name>Control Your Tabs</name>
```

File: controlyourtabs/gedit-control-your-tabs.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <summary>Gedit plugin to switch between document tabs using</summary>
```
Comment (jefferyto) [inline controlyourtabs/gedit-control-your-tabs.metainfo.xml]:
```suggestion
  <summary>A gedit plugin to switch between document tabs using Ctrl+Tab</summary>
```

File: controlyourtabs/gedit-control-your-tabs.metainfo.xml (hunk 1 above)
Comment (tim77) [reply controlyourtabs/gedit-control-your-tabs.metainfo.xml]: Indeed. That's right and much better. As for move it into `data/` may you move it yourself somehow or in next commit? Also i think email info better change to yours probably, or remove it at all. This looks more like distro specific thing, but i am not sure.

**Comment by jefferyto** [comment]
Thanks again - I've merged this into develop, will wait a little bit to see if there is any other changes I want to add before doing a release.

**Pull request merged**